core/frontend/src/SubCategoriesCache.ts

```diff
@@ -58,14 +58,25 @@ export class SubCategoriesCache {
 
       return !request.wasCanceled;
     });
-
     return {
       missingCategoryIds: missing,
       promise,
       cancel: () => request.cancel(),
     };
   }
 
+  /** Load all subcategories that come from used spatial categories of the iModel into the cache. */
+  public async loadAllSubCategories(): Promise<void> {
```

Comment:
```suggestion
  public async loadAllUsedSpatialSubCategories(): Promise<void> {
```
Make function name reflect what it actually does.